lib/omnibus/software.rb

```diff
@@ -164,6 +164,22 @@ def name(val = NULL)
     end
     expose :name
 
+    # Sets wether the software should always be built or not.
```

Comment:
we could add some advice to put the `always_build` software defs at the end of the dep list of the project to avoid always rebuilding many software